docs/source/CHANGELOG.md

```diff
@@ -7,13 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 6.0.0 - 2022-06-20
+
+### Added
+ - Added internal_index_image_copy attributes support
```

Comment:
Minor: internal_index_image_copy and internal_index_image_copy_resolved ?

Reply:
Updated.